app/controllers/bus_stops_controller.rb

```diff
@@ -11,7 +11,8 @@ def autocomplete
   def by_route
     @route = Route.find_by number: params.require(:number)
     if @route.present?
-      @stops = @route.bus_stops.order(:name)
+      @stops = @route.bus_stops.order :name
+      @inc_and_comp = @stops.group_by(&:completed?)
```

Comment:
I didn't know what to name this. I could combine line 14 and 15 into `@stops = @route.bus_stops.order(:name).group_by(&:completed?)` and fix the view accordingly.

Reply:
Great, same. I'll do that.